idp-service/README.md

```diff
@@ -1,8 +1,63 @@
 ## Identity Provider Service
 
+IdP OIDC service, provides endpoints to 
+1. Generate auth-code 
+2. Fetch id-token and access-token
+3. Fetch userinfo (KYC)
+
 ![](/docs/IdP-service-basic-flow.png)
 
 
+## Caching details
+
+IdP-UI transaction
+
+| Endpoint    | Cache                              | Evict                              |
+|-------------|------------------------------------|------------------------------------|
+| oauthDetails | preauthsessions (k: transactionId) |                                    |
+| authenticate | authenticated (k: transactionId)   | preauthsessions (k: transactionId) |
+| authCode    | consented (k: code)                | authenticated (k: transactionId)   |
+| token       | kyc   (k: accessTokenHash)         | consented  (k: code)           |
+
+
+Linked transactions
+
+| Endpoint           | Cache                                                     | Evict                              |
+|-----------------|-----------------------------------------------------------|------------------------------------|
+| oauthDetails    | preauthsessions (k: transactionId)                        |                                    |
+| linkTransaction | linkedsessions (k: linkTransactionId)                     | preauthsessions (k: transactionId) |
+| authenticate    | linkedauth (k: linkTransactionId)                 |                                    |
```

Comment:
We don't evict  anything on this, how do avoid authenticating again ?

Reply:
I still see we are not evicting anyting on authenticate, how are we handling this ?

Reply:
Updated readme.